development/webpack/webpack.integration.tests.config.ts

```diff
@@ -67,6 +69,7 @@ const config = {
                 plugins: [
                   autoprefixer({ overrideBrowserslist: browsersListQuery }),
                   rtlCss({ processEnv: false }),
+                  cssnano({ preset: 'default' }),
```

Comment:
Updating webpack configuration for integration tests(`yarn test:integraion`) to minify CSS using `cssnano`